partner-api.yaml

```diff
@@ -485,6 +485,15 @@ components:
         - $ref: '#/components/schemas/Plakette'
         - type: object
           properties:
+            parent:
+              type: object
+              title: Parent
```

Comment:
Der `title` ist in allen anderen Objekten klein geschrieben. Warum hier jetzt Großschreibung?